src/components/PeopleDropdown/PeopleDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useRef, useState } from 'react';
+import { PeopleList } from '../PeopleList';
+import { Person } from '../../types/Person';
+
+function debounce(
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  callback: Function,
+  delay: number,
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  afterCallback: Function,
+) {
+  let timerId = 0;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return (...args: any) => {
+    window.clearTimeout(timerId);
+
+    timerId = window.setTimeout(() => {
+      callback(args[0]);
+      afterCallback(args[1]);
+    }, delay);
+  };
+}
+
+type Props = {
```

Comment:
use interface prefer.
Types have unique features but the interface is best for scaling.